var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -118,6 +120,21 @@ def setup_dependent_package(self, module, dep_spec):
         self.spec.mpifc = join_path(self.prefix.bin, 'mpif90')
         self.spec.mpif77 = join_path(self.prefix.bin, 'mpif77')
 
+    def setup_environment(self, spack_env, run_env):
+        # As of 06/2016 there is no mechanism to specify that packages which
+        # depends on MPI need C or/and Fortran implementation. For now
+        # require both.
+        if (self.compiler.f77 is None) or (self.compiler.fc is None):
+            tty.warn('OpenMPI : FORTRAN compiler not found')
+            tty.warn('OpenMPI : FORTRAN bindings will be disabled')
```

Comment:
Does this work? In the past when I've tried to use tty in a package it printed to `spack-build.out` instead of STDOUT.